src/components/Navbar.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from "react-router-dom";
+
+export const Navbar = () => {
+  return (
+    <header className="header fixed-header">
+      <nav className="navbar row-flex w-95p m-auto p-05">
+        <a className="m-l-3" href="/">
+          <h3 id="logo">
+            in.notion <span className="text-shd">Care</span>
+          </h3>
+        </a>
+        <ul className="nav-items row-flex w-40p no-bullet">
+          <li className="btn nav-link-btn cursor h3">
+            <a href="pages/products.html">Shop</a>
+          </li>
+          <li className="btn nav-link-btn cursor h3">
+            <a href="pages/products.html">Care products</a>
+          </li>
+          <li className="btn nav-link-btn cursor h3">
+            <a>About</a>
+          </li>
+        </ul>
+        <label className="field searchfield w-20p" htmlFor="search-text">
+          <span className="search-icon cursor p-h-1">
+            <i className="fa fa-search" aria-hidden="true"></i>
+          </span>
+          <input type="search" className="input search-nav reset-ip p-05" placeholder="Search here" id="search-text" />
+        </label>
+        <div className="nav-icon-btns row-flex no-wrap">
+          <ul className="row-flex no-bullet">
+            <li className="nav-icon-btn icon-btn rd-bdr grid-ctr wt-text m-r-3">
```

Comment:
Here, these Lists are repeating. Maybe you can think to loop over an array and then create these lists.

Reply:
Done with the changes. I've added Navbar & footerData for looping.